docs/faq.txt

```diff
@@ -214,10 +214,10 @@ container image that has been configured for that experiment. Determined
 provides prebuilt Docker images that include TensorFlow 1.15, 2.4, and
 2.5, respectively:
 
--  ``determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-0.16.0``
+-  ``determinedai/environments:cuda-10.2-pytorch-1.7-tf-1.15-gpu-0.16.1``
    (default)
```

Comment:
I believe the second one is now the default. can we reorder them and update this accordingly?

Reply:
nice catch. done